.github/scripts/test-project.sh

```diff
@@ -28,6 +28,23 @@ echo ""
 echo "-----------------------------"
 echo "running $dir/$project"
 
+
+# Find schema, if it contains `env("DATABASE_URL")`, db push that schema to database
+if [[ "$project" = "planetscale" ]]
+then
+  true
+  # have to skip planetscale until https://github.com/planetscale/cli/issues/322 is implemented
+else
+  schema_path=$(find $dir/$project -name "schema.prisma" ! -path "*/node_modules/*" | head -n 1)
+  if grep -q "env(\"DATABASE_URL\")" "$schema_path"; then
+    echo ""
+    echo "found 'schema.prisma' with 'env(\"DATABASE_URL\")': $schema_path"
+    echo "npx prisma db push --accept-data-loss --skip-generate --schema=$schema_path"
+    npx prisma db push --accept-data-loss --skip-generate --schema=$schema_path
```

Comment:
So here npx prisma should run latest I think? Since the root package has no prisma dependency.

Reply:
Yep, using Migrate here via `npx prisma` is generally a tradeoff of this approach. If that ever breaks, the whole tests are in trouble. Not sure how to solve that, as we removed the top level dependency for a reason.

Reply:
A few thoughts for solutions:
* Maybe we could `cd` into the directory before running `npx`.
* Or a *special* folder with a `package.json` where we run that `npx`.